Lib/test/test_functools.py

```diff
@@ -1866,6 +1866,7 @@ def orig(): ...
 
     @support.skip_on_s390x
     @unittest.skipIf(support.is_wasi, "WASI has limited C stack")
+    @unittest.skipIf(support.PGO, "PGO builds may have reduced C stack")
```

Comment:
Since this only fails on Windows, should we skip the test on all platforms?

Reply:
Turns out there are other tests that cause a crash, so I'm going to revert this anyway and try tweaking a build setting.